src/avfoundation.cs

```diff
@@ -2507,6 +2789,9 @@ interface AVAsset : NSCopying {
 		[Export ("metadataForFormat:")]
 		AVMetadataItem [] MetadataForFormat (string format);
 
+		[Wrap ("MetadataForFormat (format.GetConstant ())")]
+		AVMetadataItem [] MetadataForFormat (AVMetadataFormat format);
```

Comment:
Let's rework this so we
1. expose `GetMetadata` for `NSString` (not `string`) with `[Advanced]`
2. put an `[Obsolete]` on the old/badly named API and put it under `if #XAMCORE_4_0`
3. Have the Wrap version call the non-obsolete API